### PR TITLE
arpack: migrate from homebrew/science

### PR DESCRIPTION
**Pull request by fxcoudert**
See https://github.com/Homebrew/homebrew-core/issues/15619

### CODE REVIEW

**Comment by fxcoudert** [comment]
@BrewTestBot test this please

File: Formula/arpack.rb [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    end
+
+    system "./bootstrap"
+    system "./configure", *args
+    system "make"
+    system "make", "install"
+
+    lib.install_symlink Dir["#{libexec}/lib/*"].select { |f| File.file?(f) }
+    (lib/"pkgconfig").install_symlink Dir["#{libexec}/lib/pkgconfig/*"]
+    (libexec/"share").install "TESTS/testA.mtx", "TESTS/dnsimp.f",
```
Comment (ilovezfs) [inline Formula/arpack.rb]: probably should use pkgshare

File: Formula/arpack.rb [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+
+  depends_on :fortran
+  depends_on "veclibfort"
+  depends_on :mpi => [:optional, :f77]
+
+  def install
+    args = [ "--disable-dependency-tracking",
```
Comment (ilovezfs) [inline Formula/arpack.rb]: args = %W[
]

Comment (fxcoudert) [inline Formula/arpack.rb]: I'm not sure how to quote the last argument, with the space in it, if I used `%W[]` :(

Comment (fxcoudert) [inline Formula/arpack.rb]: Found it, simply `\ `. Less readable in my opinion, but if you want it…

**Pull request closed without merging**